packages/rum-core/src/performance-monitoring/metrics.js

```diff
@@ -269,3 +373,21 @@ export class PerfEntryRecorder {
     this.po.disconnect()
   }
 }
+
+export function bootstrapMetrics() {
+  metrics.wasHidden = document.visibilityState === 'hidden'
+
+  window.addEventListener(
+    'visibilitychange',
+    () => {
+      if (document.visibilityState === 'hidden') {
+        metrics.wasHidden = true
```

Comment:
Did you check about the use of timestamp?

Reply:
We decided to use visibiltyState instead of responsiveness in a separate PR.